demos/browser/package.json

```diff
@@ -29,8 +29,7 @@
     "ts-loader": "^8.1.0",
     "typescript": "^4.2.3",
     "webpack": "^4.44.2",
-    "webpack-cli": "^4.6.0",
-    "webpack-dev-server": "^3.11.2"
```

Comment:
I think webpack still needs `webpack-dev-server` or it errors out. I got this when I was trying to upgrade one of the other demo. Did you ran into this?

Reply:
It works for me without it.

Reply:
Huh, it worked, and then the second invocation prompted me to install w-d-s. Weird!